### PR TITLE
Fix margin of nested tabs and display back button on media activity tab

### PR DESCRIPTION
**Pull request by niklasnatter**

### CODE REVIEW

File: src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Admin/ActivityAdmin.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ class ActivityAdmin extends Admin
 
     const LIST_VIEW = 'sulu_activity.activities.list';
 
-    const EDIT_FORM_ACTIVITY_VERSION_TAB_VIEW = 'sulu_page.page_edit_form.activity_version_tab';
+    const PAGE_EDIT_FORM_ACTIVITY_TAB_VIEW = 'sulu_page.page_edit_form.activity_version_tab';
```
Comment (niklasnatter) [inline src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Admin/ActivityAdmin.php]: @alexander-schranz should we move these views into the `PageAdmin`/`MediaAdmin` or should we leave them here? 🤔

Comment (niklasnatter) [reply src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Admin/ActivityAdmin.php]: not only the constant but also the `$viewCollection->add()` calls

Comment (alexander-schranz) [reply src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Admin/ActivityAdmin.php]: They should exist in `PageAdmin` and `MediaAdmin` or be there own `PageActivityAdmin` and  `PageMediaAdmin` classes but still in there specific bundles

Comment (alexander-schranz) [reply src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Admin/ActivityAdmin.php]: I think this can be removed now?

**Pull request merged**